provider/cmd/pulumi-tfgen-frontegg/pulumi-tfgen-xyz/main.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016-2018, Pulumi Corporation.
```

Comment:
I think the directory name should likely be `pulumi-tfgen-frontegg`

Reply:
oh actually it looks like that already exists and this is an extra dir/file that's not needed